interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -208,463 +140,156 @@ Specification
-
-Preconditions
-.............
-
-* The BTC Parachain status in the :ref:`security` component must be set to ``RUNNING:0``.
-
-
-Function Sequence
-.................
-
-1. Check ``operatorId`` is an operator. If not, throw ``VaultNotOptedInToNomination``.
-
-2. Check ``requestId`` corresponds to an actual withdrawal request. If not, throw ``WithdrawalRequestNotFound``.
-
-3. Remove the withdrawal request from the ``withdrawalRequests`` array in the ``Operator`` struct for ``operatorId``.
-
-4. Increase the backing collateral of ``operatorId`` in the :ref:`Vault-registry` by the amount in the withdrawal request.
-
-
-.. _cancelNominatorWithdrawal:
-
-cancelNominatorWithdrawal
--------------------------
+withdrawCollateral
+------------------
 
-Cancel a nominator's withdrawal request.
+Withdraw collateral from a nominated Vault.
 
 Specification
 .............
 
 *Function Signature*
 
-``cancelNominatorWithdrawal(nominatorId, operatorId, requestId)``
+``withdrawCollateral(vaultId, nominatorId, amount)``
 
 *Parameters*
 
-* ``nominatorId``: the id of the nominator.
-
-* ``operatorId``: the id of the operator.
-
-* ``requestId``: the id of the withdrawal request.
+* ``vaultId``: the id of the previously nominated Vault.
+* ``nominatorId``: the id of the user who nominated collateral.
+* ``amount``: the amount of collateral to withdraw.
 
 *Events*
 
-* ``CancelNominatorCollateralWithdrawal(requestId, nominatorId, operatorId)``
-
-*Errors*
-
-* ``VaultNotOptedInToNomination``: the ``operatorId`` is not an Operator.
-
-* ``NominatorNotFound``: the ``nominatorId`` is not a Nominator.
-
-* ``WithdrawalRequestNotFound``: no withdrawal request found for the given id.
-
-Preconditions
-.............
-
-* The BTC Parachain status in the :ref:`security` component must be set to ``RUNNING:0``.
-
-
-Function Sequence
-.................
+* ``WithdrawCollateral(vaultId, nominatorId, amount)``
 
-1. Check ``operatorId`` is an operator. If not, throw ``VaultNotOptedInToNomination``.
+*Preconditions*
 
-2. Check ``nominatorId`` is a nominator. If not, throw ``NominatorNotFound``.
+* The BTC Parachain status in the :ref:`security` component MUST be ``RUNNING:0``.
+* The global nomination flag MUST be enabled.
+* A Vault with id ``vaultId`` MUST be registered.
+* A Vault with id ``vaultId`` MUST exist in the ``Vaults`` mapping.
+* The Vault MUST remain above the secure collateralization threshold.
+* Nominator MUST have nominated at least `amount`.
 
-3. Check ``requestId`` corresponds to an actual withdrawal request. If not, throw ``WithdrawalRequestNotFound``.
+*Postconditions*
 
-3. Remove the withdrawal request from the ``withdrawalRequests`` array in the ``Nominator`` struct inside the ``Operator`` struct for ``operatorId``.
-
-4. Increase the backing collateral of ``operatorId`` in the :ref:`Vault-registry` by the amount in the withdrawal request.
+* The Vault's collateral MUST decrease by the amount nominated.
```

Comment:
I guess also here the nominators locked collateral with that vault must decrease by amount?

Reply:
Would also reference this in the `staking` specification.